packages/ddp-client/common/livedata_connection.js

```diff
@@ -1593,9 +1588,10 @@ export class Connection {
     }
 
     // Now add the rest of the original blocks on.
-    oldOutstandingMethodBlocks.forEach(block => {
-      self._outstandingMethodBlocks.push(block);
-    });
+    self._outstandingMethodBlocks = [
+      ...self._outstandingMethodBlocks,
+      ...oldOutstandingMethodBlocks
+    ];
```

Comment:
Let's do `self._outstandingMethodBlocks.push(...oldOutstandingMethodBlocks)` here, to avoid creating a new array.

Reply:
ok, I updated it!